random_graphs.py

```diff
@@ -51,45 +51,6 @@ def make_complete_graph(n):
             g[j].append(i)
     return g
 
-def make_random_graph(n, density, seed = None, max_degree = np.inf, min_degree = 0):
```

Comment:
Removed the `make_random_graph` function since the `get_random_connected_graphs` function replaces it.